docs/index.rst

```diff
@@ -148,6 +148,8 @@ tedana is licensed under GNU Lesser General Public License version 2.1.
    usage
    approach
    outputs
+   support
```

Comment:
total nitpick, but could we flip the order, here ? I'd like folks to read the FAQ before posting on Neurostars :smile_cat:

Reply:
:+1: